backend/src/modules/prisma.ts

```diff
@@ -0,0 +1,33 @@
+import { PrismaClient as Connection } from "@prisma/client";
+import { getLogger } from "../utils/logger";
+import env from "../env";
+
+export default class PrismaClient {
+  private logger = getLogger();
+  private connection: Connection;
+
+  constructor() {
+    this.connection = new Connection({
+      // log: ["query"],
```

Comment:
would be good for debugging to get logs of the output sql queries when locally developing

Reply:
yes but surely we dont need permanent debugging no??

Reply:
we dont want prod logs for this

Reply:
hmm im not too sure what ur saying then 🤔 what exactly do you think should get logged

Reply:
`log: process.env.NODE_ENV !== "production" && ["query"]`